packages/components/src/components/drawer/drawer.ts

```diff
@@ -156,12 +136,12 @@ export default class SdDrawer extends SolidElement {
     document.removeEventListener('keydown', this.handleDocumentKeyDown);
   }
 
-  private handleDocumentKeyDown(event: KeyboardEvent) {
+  private handleDocumentKeyDown = (event: KeyboardEvent) => {
     if (this.open && !this.contained && event.key === 'Escape') {
```

Comment:
```suggestion
    if (this.open && && event.key === 'Escape') {
```

Unfortunately we have to do this, as otherwise we can't close it via keyboard in the stories.